.github/workflows/e2e-autogen-internals.yaml

```diff
@@ -24,7 +24,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        k8s-version: [v1.21.12, v1.22.9, v1.23.6, v1.24.0]
+        k8s-version: [v1.21.14, v1.22.13, v1.23.10, v1.24.4, v1.25.0]
```

Comment:
Should we drop 1.21.X?

Reply:
I think so, shall I drop it in this PR ?

Reply:
Yes, let's drop it. I'll update the branch protection then to get this merged.

Reply:
Fixed